### PR TITLE
Bug 1168876 - Failure in test_system_notification_bar.py, caused by pull request from bug 1164073

### PR DESCRIPTION
**Pull request by mwargers**

### CODE REVIEW

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=c1eeaf8070b98b4c9d50992b5edc3f9010106b4a

File: tests/python/gaia-ui-tests/gaiatest/tests/functional/system/test_system_notification_bar.py [hunk 1]
```diff
@@ -19,7 +19,7 @@ def test_notification_bar(self):
         self.marionette.execute_script('new Notification("%s", {body: "%s"});'
                                        % (self._notification_title, self._notification_body))
 
-        system.wait_for_notification_toaster_displayed(for_app='system')
+        system.wait_for_notification_toaster_displayed()
```
Comment (mwargers) [inline tests/python/gaia-ui-tests/gaiatest/tests/functional/system/test_system_notification_bar.py]: Oops, this doesn't need to be reverted

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=5e657b4338931776d24fe712ecd33564df469ace

File: tests/python/gaia-ui-tests/gaiatest/tests/functional/system/test_system_notification_bar.py [hunk 2]
```diff
@@ -29,9 +29,12 @@ def test_notification_bar(self):
 
         utility_tray.wait_for_notification_container_displayed()
 
+        # Assert there is one system notification is listed in notifications-container
         notifications = utility_tray.get_notifications(for_app='system')
-        self.assertEqual(1, len(notifications), 'Expected one system notification.')
-        email = notifications[0].tap_notification()
+        self.assertEqual(1, len(notifications), 'Expected one notification.')
```
Comment (JohanLorenzo) [inline tests/python/gaia-ui-tests/gaiatest/tests/functional/system/test_system_notification_bar.py]: Why do we change the error message? It seems like we still wait for a system notification to be displayed.

File: tests/python/gaia-ui-tests/gaiatest/tests/functional/system/test_system_notification_bar.py [hunk 3]
```diff
@@ -29,9 +29,12 @@ def test_notification_bar(self):
 
         utility_tray.wait_for_notification_container_displayed()
 
+        # Assert there is one system notification is listed in notifications-container
```
Comment (JohanLorenzo) [inline tests/python/gaia-ui-tests/gaiatest/tests/functional/system/test_system_notification_bar.py]: I don't think this comment add some value to the lines below.

File: tests/python/gaia-ui-tests/gaiatest/tests/functional/system/test_system_notification_bar.py [hunk 4]
```diff
@@ -29,9 +29,12 @@ def test_notification_bar(self):
...
         notifications = utility_tray.get_notifications(for_app='system')
-        self.assertEqual(1, len(notifications), 'Expected one system notification.')
-        email = notifications[0].tap_notification()
+        self.assertEqual(1, len(notifications), 'Expected one notification.')
+
+        # Assert notification is listed in notifications-container
```
Comment (JohanLorenzo) [inline tests/python/gaia-ui-tests/gaiatest/tests/functional/system/test_system_notification_bar.py]: Same thing about this comment

File: tests/python/gaia-ui-tests/gaiatest/tests/functional/system/test_system_notification_bar.py (hunk 2 above)
Comment (mwargers) [inline tests/python/gaia-ui-tests/gaiatest/tests/functional/system/test_system_notification_bar.py]: Good catch.

File: tests/python/gaia-ui-tests/gaiatest/tests/functional/system/test_system_notification_bar.py (hunk 3 above)
Comment (mwargers) [inline tests/python/gaia-ui-tests/gaiatest/tests/functional/system/test_system_notification_bar.py]: I'll remove the comments.

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=224b8d20d72dde747af0ede41230c0558b35816c

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=5aff7fa61a1c629e1dbc52c66eb061a4fc1f6aad

**Pull request merged**